sssom/cli.py

```diff
@@ -564,7 +564,7 @@ def correlations(input: str, output: str, transpose: bool, fields: Tuple):
 
     tups = []
     for i, row in corr.iterrows():
-        for j, v in row.iteritems():
+        for j, v in row.items():
```

Comment:
Is this save? Why that change?

Reply:
iteritems() is not necessarily forwards compatible with python3.